### ISSUE TITLE
Documentation needs updating for Custom map functions

### ISSUE THREAD

**Issue by jackie-h** [jtablesaw/tablesaw#694]
https://jtablesaw.github.io/tablesaw/userguide/mapping

doWithEach() no longer exists. Is map() the replacement?

**Comment by benmccann** [comment]
I don't see `doWithEach` being removed in that commit.

It depends what you're trying to accomplish. If you want to create a new column like in the documentation, then yes, `map()` would be great.

If you just want to use the values an not create anything new then `.asList().stream().forEach()` might be one way to do it

We'd love a PR fixing the docs if you'd like to send one! We have some new infrastructure to test the code in the docs, but that page hasn't been updated yet

**Comment by jackie-h** [comment]
Thanks for the reply. The commit was https://github.com/jtablesaw/tablesaw/pull/331/commits/2da0134d255aae96230013f3b63a61521d20e906
I had pasted it incorrectly and then removed it above.

Happy to have a go at updating the docs, but I'm currently having trouble making it work. I tried with map(), but it doesn't let me append the column, as it thinks it is a duplicate. That makes sense, as the column has the same name.
